docs/source/installation/win.rst

```diff
@@ -1,2 +1,95 @@
 Windows
 =======
+
+Cairo installation
+******************
+
+1. Visit the official `Download Page
+   <https://www.lfd.uci.edu/~gohlke/pythonlibs/#pycairo>`_.
+
+2. Select the download that corresponds to your PC's architechture and Python
+   Version
+
+   .. image:: ../_static/installation/windows_cairo.png
+       :align: center
+       :width: 400px
+       :alt: windows cairo download page
+
+   .. note:: ``cp38`` corresponds to Python 3.8, ``cp37`` corresponds to Python
+             3.7, etc. ``win_amd64`` corresponds to 64-bit machines, ``win32``
+             corresponds to 32-bit machines.
+
+3. Open up your command prompt by hitting the Win key and typing ``cmd``.
+
+4. Enter the directory where you want to install cairo by typing ``cd
+   C:\path\to\cairo`` with the path being where you downloaded the ``.whl``
+   file in step 2.
+
+5. Finally, run ``pip3 install (cairo-file).whl``, where the file the
+   appropriate name of the ``.whl`` file.
+
+6. Alternatively, running the command below installs pycairo.  This needs to be
+   an elevated command prompt like Powershell.
+
+   .. code-block:: bash
+
+      (Invoke-WebRequest -Uri https://raw.githubusercontent.com/ManimCommunity/manim/master/scripts/pycairoinstall.py -UseBasicParsing).Content | py -3
+
+
+FFmpeg installation
+*******************
+
+1. To install ``ffmpeg`` and add it to your PATH, install `Chocolatey
+   <https://chocolatey.org/>`_ and run ``choco install ffmpeg``.
+
+2. You can check you did it right by running ``refreshenv`` to update your
+   environment variable and running ``ffmpeg``.
+
+
+Sox installation (optional)
+***************************
+
+1. Go to the SoX `SourceForge
```

Comment:
May be `choco install sox.portable`?

Reply:
There's no guarantee that the user will have `choco` installed. Maybe we could add that to the install instructions?

Reply:
What about ffmpeg installation?

Reply:
Okay, so I think the instruction to install `choco` should be moved to its own separate section (like the homebrew installation section in the MacOS instructions), and then we can use `choco install` where possible

Reply:
Probably we should redirect them to the site which install choco and we should not worry about updating instructions.
Redirect them here https://chocolatey.org/install